GNUmakefile

```diff
@@ -330,7 +335,7 @@ static-assets:
 ui: ui-docker static-assets-docker
 
 tools:
-	go get -u -v $(GOTOOLS)
+	go get -v $(GOTOOLS)
```

Comment:
(not blocking) I think this means you can get rid of my `git reset` in the circleci config file (but if you plan on doing that double check).

Reply:
Unfortunately it doesn't completely fix the problem either. Running make tools still updates things:

```
diff --git a/go.mod b/go.mod
index 4e48a62e2..c2d553791 100644
--- a/go.mod
+++ b/go.mod
@@ -17,12 +17,12 @@ require (
        github.com/coredns/coredns v1.1.2
        github.com/digitalocean/godo v1.10.0 // indirect
        github.com/docker/go-connections v0.3.0
-       github.com/elazarl/go-bindata-assetfs v0.0.0-20160803192304-e1a2a7ec64b0
+       github.com/elazarl/go-bindata-assetfs v1.0.0
        github.com/envoyproxy/go-control-plane v0.8.0
        github.com/go-ole/go-ole v1.2.1 // indirect
        github.com/gogo/googleapis v1.1.0
        github.com/gogo/protobuf v1.2.1
-       github.com/golang/protobuf v1.3.1
+       github.com/golang/protobuf v1.3.2
        github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf
        github.com/hashicorp/consul/api v1.2.0
        github.com/hashicorp/consul/sdk v0.2.0
@@ -77,6 +77,7 @@ require (
        golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a
        golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
        google.golang.org/grpc v1.23.0
+       gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
        gopkg.in/square/go-jose.v2 v2.3.1
        k8s.io/api v0.0.0-20190325185214-7544f9db76f6
        k8s.io/apimachinery v0.0.0-20190223001710-c182ff3b9841
```

It just updates far fewer of them.

Reply:
This might be a go modules bug fixed in Go 1.13 though as there were issues reported in the past where go get would update too many things when operating in module mode.